sigs.yaml

```diff
@@ -1386,16 +1386,23 @@ sigs:
       - https://raw.githubusercontent.com/kubernetes/features/master/OWNERS
   - name: Release
     dir: sig-release
-    charter_link:
+    charter_link: https://git.k8s.io/sig-release/charter.md
```

Comment:
/cc @timothysc @spiffxp
I don't know if there are specific guidelines for charter location, but CC'ing steering to ensure this is cool with them.

Reply:
The charter typically lives in the community repo next to all the other sig details, not in an external repo.

Reply:
all of the important documentation for SIG release lives at `github.com/kubernetes/sig-release` we are unique in that regard but I would strongly like to continue in that direction. I would say that we shouldn't use a vanity URL in the link though.

Reply:
Hmm... we provide the field to specify `charter_link`. I feel that implies that charters don't necessarily need to live in k/community as `charter.md`. If that's not the case, we should remove that field from the generator.

As Caleb mentioned, `k/sig-release` is the canonical source for all SIG Release information, so I think having the SIG's defining document adjacent to other SIG Release info is important. 
While I can get the need of not wanting break current convention, SIG Release is one of the few (maybe only) SIGs that has their own repo, and it's well-organized. Let me know what you think!

Reply:
Also, any reason to not use the vanity URL, @calebamiles?
It's the suggested method (for markdown, at least): https://github.com/kubernetes/community/blob/master/sig-contributor-experience/markdown-link-style-guide.md

Reply:
Charters should really be homed in a single location from a discoverability and policy enforcement perspective.  If there are bulk edits to how we do charters, the person responsible would have to track across (N).  I don't think sig-release is unique from a charter perspective.

Reply:
I do think SIG Release is unique insofar as where the SIGs documentation is constantly put. That said, I don't have a strong opinion on where we land our governing document, outside of saying I hope it would stay close to the day-to-day docs we craft.

If Steering would prefer the charter remain in the community repo, as opposed to in k/sig-release, I'd want that to be explicitly said, as well as having `charter_link` be removed from the generator template. If we're saying that charters will always be within the SIG dir, named as `charter.md`, than the additional field has nonuse outside of allowing confusion.

Reply:
@justaugustus The field allowed for transition when some charter docs (before there was formal structure) were in docs and the like. I don't think we need to force a technical limitation on the generator here, when a policy one will do just fine. Setting that value to `charter.md` like most other sigs have works just fine.